src/vl-map-action.js

```diff
@@ -23,18 +32,27 @@ export class VlMapAction extends vlElement(HTMLElement) {
   }
 
   /**
-   * Geeft de event naam die gebruikt wordt wanneer een nieuwe actie toegevoegd wordt aan de kaart
+   * Geeft de stijl van de kaartinteractie.
```

Comment:
Wat is juist de kaartinteractie? Denk dat het eerder 'kaart actie' is dan, dan is het consistent met documentatie in action() hieronder